cli/script/command-executor.ts

```diff
@@ -1326,8 +1326,8 @@ export const releaseReact = (command: cli.IReleaseReactCommand): Promise<void> =
           ? Q(command.appStoreVersion)
           : getReactNativeProjectAppVersion(command, projectName);
 
-        if (command.outputDir) {
-          command.sourcemapOutput = path.join(command.outputDir, bundleName + ".map");
+        if (command.sourcemapOutput) {
```

Comment:
Can we check if the `sourcemapOutput` already ends with `.map`? In that case we don't need adding it manually.

Reply:
Thank you for pointing that out! I've updated it.